prov/net/src/xnet_init.c

```diff
@@ -137,6 +138,10 @@ static void xnet_init_env(void)
 			"prevent auto-progress thread from starting");
 	fi_param_get_bool(&xnet_prov, "disable_auto_progress",
 			&xnet_disable_autoprog);
+	fi_param_define(&xnet_prov, "uring", FI_PARAM_INT,
+			"Enable uring support, set to 1 to enable");
+	fi_param_get_int(&xnet_prov, "uring",
+			 &xnet_uring);
```

Comment:
I will change `configure.ac` to always build with libuing if the library is present on the system (as you suggested), but will disable it by default via the environment variable.
FYI, I intentionally wanted to disable io_uring by default because its support is known to be bogus with certain older versions of the Linux kernel.